packages/styles/scss/patterns/blocks/feature-card-block-large/_feature-card-block-large.scss

```diff
@@ -0,0 +1,147 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+@import '../../../components/image/image';
+@import '../../sub-patterns/card/index';
+@import '../../../globals/utils/aspect-ratio';
+@import '../../../globals/utils/hang';
+
+$fcb-large-custom-bp-copy: 992px;
+$fcb-large-custom-bp-nocopy: 752px;
+
+@mixin feature-card-block-large {
+  .#{$prefix}--feature-card-block-large {
+    color: $text-01;
+    background: $ui-background;
+    position: relative;
+    padding-top: 20px;
+    padding-bottom: 20px;
+    height: 100%;
+
+    @include carbon--breakpoint-between($fcb-large-custom-bp-copy, 'lg') {
+      padding-top: aspect-ratio(2, 1);
+    }
+    @include carbon--breakpoint('xlg') {
+      padding-top: aspect-ratio(2, 1);
+    }
+
+    &__card.#{$prefix}--card--inverse .#{$prefix}--card__cta,
+    &__card.#{$prefix}--card--inverse .#{$prefix}--card__cta:visited {
+      float: none;
+      color: $inverse-link;
+      font-size: 2rem;
+    }
+
+    .#{$prefix}--card {
+      background: none;
+      height: 100%;
+
+      @include carbon--breakpoint-between($fcb-large-custom-bp-copy, 'lg') {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        flex-direction: row;
+      }
+      @include carbon--breakpoint('xlg') {
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        flex-direction: row;
+      }
+
+      &__wrapper,
+      .#{$prefix}--image {
+        @include carbon--breakpoint-between($fcb-large-custom-bp-copy, 'lg') {
+          width: 50%;
+          height: 100%;
+        }
+        @include carbon--breakpoint('xlg') {
+          width: 50%;
+          height: 100%;
+        }
+      }
+
+      .#{$prefix}--image {
+        overflow-y: hidden;
+      }
+
+      &__wrapper {
+        background: $inverse-02;
+        min-height: 50%;
+        height: auto;
+
+        @include carbon--breakpoint('xlg') {
+          padding: rem(32px);
+        }
+
+        @include carbon--breakpoint('max') {
+          padding: rem(64px);
+        }
+
+        .#{$prefix}--card__eyebrow {
+          margin-bottom: rem(14px);
+          color: #ffffff;
```

Comment:
There should be a token for this one too...

Reply:
Looks like eyebrow uses

`color: $inverse-01`

Reply:
the color i've fixed, so i want to check if it's for the margin-bottom